config/oss/bucket/config.go

```diff
@@ -0,0 +1,13 @@
+package oss
+
+import "github.com/crossplane/terrajet/pkg/config"
+
+// Configure configures individual resources by adding custom ResourceConfigurators.
+func Configure(p *config.Provider) {
+	p.AddResourceConfigurator("alicloud_oss_bucket", func(r *config.Resource) {
+
+		// we need to override the default group that terrajet generated for
+		// this resource, which would be "github"
+		r.ShortGroup = "oss"
```

Comment:
fwiw, this custom configuration is probably not needed and the default would already be `oss` in this case different than the tutorial.

Reply:
Let me keep this line for a while. After removing it, many files are to be removed and more are to be added with `make reviewable`. Please allow me more time to figure it out and I will remove them next time.